resources/test/sse_data_schema.json

```diff
@@ -3348,6 +3349,12 @@
           "type": "integer",
           "format": "uint64",
           "minimum": 0.0
+        },
+        "whitelist_size": {
```

Comment:
we should not call this a whitelist, instead call it a reserve list

Reply:
Renamed the field to `reserved_spots`.